### ISSUE TITLE
Preview Tab only shows default 2000m by 2000m course areas.

### ISSUE THREAD

**Issue by jrobbb** [chadrockey/TGC-Designer-Tools#41]
when my course is created and i go to the preview section, edges on the west and east are cut off (including portions of a few holes).  I made sure to select a wide enough area when importing all lidar data.  so not sure what's causing it to be cut off, unless there is an inherent area limit baked in.

**Comment by chadrockey** [comment]
This is probably because the preview tab in the tools was coded for the default 2000m x 2000m bounding box.  If you export to in game, it should all work.

I can't promise that more than 2000m x 2000m will always work into the future, but it does for now.

Just load into game and see what it looks like.

**Comment by jrobbb** [comment]
Thanks. The preview actually was fairly true to what was shown in game. I
was able to work around it by rotating the course as it fit better at an 45
degree angle.

Such an incredible tool you created.  Really have enjoyed using it and am
continued to be amazed at how accurately it creates courses.

**Comment by chadrockey** [comment]
You shouldn’t have had to do that.  Did you forget to add line-> holes on OSM?  I don’t like the rotation stuff anymore because it messes with the sun and shadows.

**Comment by jrobbb** [comment]
I did add the holes in OSM (another awesome feature). I think the area of
the course (from east to west) was just really really wide.  It's only
cutting off about 30 % of the hole on the far edge of the course.

I've created four courses with the tool and it was just that one course
that had the issue.

Maybe I'll paint an extra hole on the edge of the map file to try and trick
it into making the course area wider.
